app/src/main/res/layout/fragment_add_donor.xml

```diff
@@ -124,5 +132,5 @@
         android:indeterminate="true"
         android:paddingTop="10dp"
         app:indicatorColor="@color/red"/>
-
-</LinearLayout>
+    </LinearLayout>
```

Comment:
Should the circular progress indicator be outside of the linear layout? That way it can be centered in the middle of the view

Reply:
done

Reply:
this looks like the progress view is still in the linear layout